iosApp/iosApp/Pages/Search/SearchResultsContainer.swift

```diff
@@ -0,0 +1,382 @@
+//
```

Comment:
This is a lot of top-level structs for a single file, I think. What do you think of breaking this up a bit?

Reply:
I agree! I'll go ahead and break it up.